### PR TITLE
New CCD Camera Python Example

### PR DESCRIPTION
**Pull request by manzelmi-thorlabs**
Adding a new CCD python example that uses the .NET camera SDK with the pythonNET package. Library is included.

### CODE REVIEW

File: Python/Obsoleted/CCD Camera PythoNET/tl_dotnet_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+clr.AddReference('Thorlabs.TSI.ImageDataInterfaces')
+clr.AddReference('Thorlabs.TSI.ColorInterfaces')
+clr.AddReference('CameraTypeConversion')
+
+import Thorlabs.TSI.ImageData as thorlabs_tsi_imagedata
+import Thorlabs.TSI.ImageDataInterfaces 
+import Thorlabs.TSI.TLCamera as thorlabs_tsi_tlcamera
+import Thorlabs.TSI.TLCameraInterfaces as thorlabs_tsi_tlcamerainterfaces
+import Thorlabs.TSI.ColorInterfaces as thorlabs_tsi_colorinterfaces
+from CameraTypeConversion import Converter
```
Comment (wskopalik) [inline Python/Obsoleted/CCD Camera PythoNET/tl_dotnet_wrapper.py:22]: I still get an error in this line:

ImportError: cannot import name 'Converter' from 'CameraTypeConversion' (unknown location)

If I use "from CameraTypeConversion import *" or "import CameraTypeConversion", the debugging works well and there is no error. So it looks like "CameraTypeConversion" can generally be found. Just the "Converter" cannot be found.

I have tested this in IDLE and Visual Studio Code and I'm unfortunately not sure what I'm doing wrong.

File: Python/Obsoleted/CCD Camera PythoNET/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+   * 32-Bit - C:\Program Files (x86)\Thorlabs\Scientific Imaging\Scientific Camera Support\Scientific Camera Interfaces.zip
+   * 64-Bit - C:\Program Files\Thorlabs\Scientific Imaging\Scientific Camera Support\Scientific Camera Interfaces.zip
+This example will use the dll's here: \Scientific Camera Interfaces\SDK\Native Toolkit\dlls
+
+1. Copy the dll's from \Scientific Camera Interfaces\SDK\DotNet Toolkit\dlls\Managed_64_lib folder into the location where the wrapper and example are located. 
+2. Change the Solution platform to match the bit version of the dll's.
+3. Copy the CameraTypeConversion.dll to the example/wrapper location as well if not done already
+4. Open the folder containing the files in your preferred IDE. PyCharm and Visual Studio Code have been verified. 
+
```
Comment (wskopalik) [inline Python/Obsoleted/CCD Camera PythoNET/README.md:22]: The instructions are really helpful to get started.

File: Python/Obsoleted/CCD Camera PythoNET/pythonnet_example.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+camera.set_frames_per_trigger_zero_for_unlimited(0)  # continuous mode
+camera.arm()
+camera.issue_software_trigger()
+frame = None
+while frame is None:
+    frame = camera.get_pending_frame_or_null()
+image_array = camera.frame_to_array(frame) # copies image data from frame into a numpy array
+camera.disarm()
+camera.close()
+sdk.close()
```
Comment (wskopalik) [inline Python/Obsoleted/CCD Camera PythoNET/pythonnet_example.py:20]: I could not find a CCD camera in our lab so I cannot run the example code completely. But it looks fine and I think it contains all the necessary steps which are helpful for users.

**Review by TrumanMetz — APPROVED** [review]
Program builds, I imagine this error is probably just because I don't have a camera connected.

CCD python comment

**Review by wskopalik — APPROVED** [review]

**Review by TrumanMetz — APPROVED** [review]
Really like the instructions on the readme. Builds with no errors.

**Pull request merged**